grammars/php.cson

```diff
@@ -206,7 +206,7 @@
   {
     'begin':  '''(?ix)
         (?:
-          (?:^|(?<=}))\\s*(?:(abstract|final)\\s+)?(class)\\s+([a-z_\\x{7f}-\\x{7fffffff}][a-z0-9_\\x{7f}-\\x{7fffffff}]*)
+          \\b(?:(abstract|final)\\s+)?(class)\\s+([a-z_\\x{7f}-\\x{7fffffff}][a-z0-9_\\x{7f}-\\x{7fffffff}]*)
```

Comment:
I've removed complicated look-behind. As far I'm concerned, there is no case where this could backfire, but I could be wrong.